Scheduler/feature_scheduler/maintel/fbs_config_minimal_fieldsurvey.py

```diff
@@ -67,60 +88,31 @@ def get_scheduler():
             max_alt=83.0,
             shadow_minutes=2.0,
         ),
-        basis_functions.SlewtimeBasisFunction(filtername="u_02", nside=nside),
-        basis_functions.SlewtimeBasisFunction(filtername="g_01", nside=nside),
-        basis_functions.SlewtimeBasisFunction(filtername="r_03", nside=nside),
-        basis_functions.SlewtimeBasisFunction(filtername="i_06", nside=nside),
-        basis_functions.SlewtimeBasisFunction(filtername="z_03", nside=nside),
-        basis_functions.SlewtimeBasisFunction(filtername="y_04", nside=nside),
-        basis_functions.FilterLoadedBasisFunction(filternames=sequence),
+        basis_functions.SlewtimeBasisFunction(bandname="u", nside=nside),
```

Comment:
We did add the option to have the SlewtimeBasisFunction be bandname = None, then it just returns the slewtime without considering a filter change (which is what you want I think). 
See https://rubin-scheduler.lsst.io/fbs-api-basis-functions.html#rubin_scheduler.scheduler.basis_functions.SlewtimeBasisFunction

I would suggest consolidating all of these SlewtimeBasisFunctions to a single one with "None" as the bandname.